src/librustdoc/html/render/print_item.rs

```diff
@@ -2369,27 +2365,31 @@ fn render_struct_fields(
             {
                 write_str(w, format_args!("<span class=\"comment\">/* private fields */</span>"));
             } else {
-                for (i, field) in fields.iter().enumerate() {
-                    if i > 0 {
-                        w.push_str(", ");
-                    }
-                    match field.kind {
-                        clean::StrippedItem(box clean::StructFieldItem(..)) => {
-                            write_str(w, format_args!("_"));
-                        }
-                        clean::StructFieldItem(ref ty) => {
-                            write_str(
-                                w,
-                                format_args!(
-                                    "{}{}",
-                                    visibility_print_with_space(field, cx),
-                                    ty.print(cx)
-                                ),
-                            );
-                        }
-                        _ => unreachable!(),
-                    }
-                }
+                write_str(
```

Comment:
I find this code much less good compared to the current one.

Reply:
I agree. I'll try to re-write and if I can't make it nicer I'll just revert this part.